Cargo.toml

```diff
@@ -16,18 +16,18 @@ categories = ["api-bindings", "asynchronous", "web-programming"]
 # To satisfy minimal version check
 openssl = "0.10.56"
 # To satisfy minimal version check
-openssl-sys = "0.9.91"
+openssl-sys = "0.9.103"
```

Comment:
Do we need to mention other library version bumps in the changelog too?

Reply:
This is a great comment and question.

I added the change to Changelog because our PR template asks for it. I made it on purpose not full as we have not agreed the way we want to update the changelog.
I guess there are 3 ways to approach dependencies updates and changelog. 
1. Mention generic "Updated dependencies"
2. Mention only major crates updates, eg tokio, serde etc
3. List all updated dependencies (ideally an automated way, eg git-cliff or cargo-release)

I prefer either 1 or 2 with the caveat that major in terms of 2 is sometimes hard to define.

Reply:
Yes, that makes sense. It would be great to automate this so that we don't have to manually add information about dependencies which will save us a bit of time. 

I think generic message is fine but if we need to know more about a certain version update we'd have to make an extra effort and dig and find a PR that updates it. With option 2, we'll have to agree on what is a major crate. We don't have many dependencies / crates right now so it should be easy to do so. Option 3 is fully automated which  in a way is nice.

It would also be nice to have a bot that updates these frequently so we don't have to. We do have it but it doesn't seem to be frequent enough?

Reply:
What's the conclusion on this one? 
Shall we include these in changelog?

Reply:
I don't believe we've had the opportunity to discuss this yet, the sync is next week. 

My view is that we should include everything we update in the changelog. Many prominent OSS projects include each library version update in their changelogs. If we do decide to do this, let's update rusty-falcon's changelog policy (can be done in a separate PR).